mantidproject/mantid
# Engg::enables GUI to load fiile by browse

Fixes #14075 & #14245 & #14007

> - [x] merge #14186 before this

**To Test**
`Interface/Diffraction/Engineering Diffraction`
_Get initial settings from [here](https://github.com/mantidproject/mantid/pull/13501)_
#14075

Test Calibration and Focus by browsing file instead of entering the run number inside the following text box:
Calibration Tab
- Vanadium #:
- Calibration sample #:

Focus Tab
- Focus Run: Run #:
- Focus Cropped: Run #:
- Focus Texture: Run #:
#14245

Should no longer  output calibration files flagged with the "UNKNOWN" prefix.
#14007

Updates settings required in order run all the focus tests, such as basic calibration settings and valid vector string are passed through to all tests. The presenter is almost up to date however there can be certain tests that can still be integrated to test some of new features on Focus tab.
